docs/howto/catalog_exports.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Data Catalogs Export
+description: This section explains how lakeFS can integrate with external Data Catalogs via metastore update operations. 
+parent: How-To
+---
+
+# Data Catalogs Export
+
+{% include toc_2-3.html %}
+
+## About Data Catalogs Export
+
+Data Catalog Export is all about integrating external Data Warehouses (i.e AWS Athena) with lakeFS.
+
+Data Catalogs such as Hive, Glue and other store metadata for services (such as Spark, Trino and Athena). They contain metadata such as the location of the table, information about columns, partitions and much more.
+Given the native integration between Spark and SQL, it’s most common that you’ll interact with tables in Spark environment.
+
+While different systems support reading directly from lakefs (i.e Gateway endpoint `s3a://`) many systems only support direct access to blockstore (i.e S3 bucket). 
+
+With Data Catalog Exports, one can leverage the versioning capabilities of lakeFS in external data warehouses and manage tables with branches and commits. 
```

Comment:
Again, I think it's query engines - not data warehouses (I'm not an expert in the data ecosystem - so maybe @ozkatz can come up with a better term). Also, this sentence can be merged into the first sentence ("Data Catalog Export is all about...")

Reply:
I added "and query engines" because Snowflake for example is a data warehouse where we plan to add integration for their catalog while Athena is indeed a query engine.